runner/runner.go

```diff
@@ -12,6 +12,12 @@ import (
 	"k8s.io/client-go/rest"
 )
 
+const (
+	// GlobalNamespace is the namespace that steward-framework uses for globally namespaced resources,
+	// including Broker and ServiceClass
+	GlobalNamespace = "steward"
```

Comment:
Can we define this in the k8s package instead of defining here and passing it? It seems to me that keeping knowledge of k8s concepts like namespace isolated to the k8s package (and its subpackages) might be ideal.